stackslib/src/core/mod.rs

```diff
@@ -104,6 +104,8 @@ pub const BITCOIN_MAINNET_STACKS_25_BURN_HEIGHT: u64 = 840_360;
 pub const BITCOIN_MAINNET_STACKS_30_BURN_HEIGHT: u64 = 867_867;
 /// This is Epoch-3.1, activation height proposed in SIP-029
 pub const BITCOIN_MAINNET_STACKS_31_BURN_HEIGHT: u64 = 875_000;
+/// This is Epoch-3.2, activation height proposed in SIP-031
+pub const BITCOIN_MAINNET_STACKS_32_BURN_HEIGHT: u64 = 907_740;
```

Comment:
Similar note here: I think this should be u64::max until the sip-031 code is all merged.